src/Stratis.Bitcoin.Features.Miner.Tests/PowMiningTest.cs

```diff
@@ -205,23 +210,6 @@ public void GenerateBlocks_SingleBlock_ReturnsGeneratedBlock()
             Assert.Equal(callbackBlock.GetHash(), blockHashes[0]);
         }
 
-        [Fact]
-        public void GenerateBlocks_SingleBlock_InvalidPreviousTip_ContinuesExecution_ReturnsGeneratedBlock()
```

Comment:
This test isnt necessary as it is covered by`GenerateBlocks_SingleBlock_MaxTriesReached_StopsGeneratingBlocks_ReturnsEmptyList`